### ISSUE TITLE
The current position of the bar with the + mark interferes / occupies the free space for the thumbnails in the group.

### ISSUE THREAD

**Issue by GitHubro5** [photodiode/panorama-view#51]
Hi.

**The "Add new group bar" does not need to be fixed on the far right.**
Despite the space on the right, up to four thumbnail images can be displayed for each row.
Probably because of the location of the "Add new group bar".
In other words, the balance between right and left is not correct at this time.
Is there any way to make the position of "Add new group bar" adjustable?

As shown in the attached photo, if you move the "Add new group bar" (Red color) to the top and then extend the rightmost line (Green color) to the end, there will be more room (Blue color) for one thumbnail image.

Note: Attached photo have been tested with a maximum group size.
In other words, if "Add new group bar" goes to the top, then "Group title bar" will be placed under it.

![a feedback about add new group bar position](https://user-images.githubusercontent.com/14139205/38538405-a54ce40c-3cce-11e8-9d1c-22c5792043f8.png)

**Comment by GitHubro5** [comment]
Here are some test result.

For reference, I tested with a maximum of a single group size.
However, arbitrarily adjusting the group size affected only the number of tabs needed to solve this problem.
Also, it depends on the currently set desktop resolution size.
Therefore, the number of tabs listed below is my standard.
I am using a 19 inch monitor with a Windows desktop resolution of 1280 x 1024.

Based on my resolution, this issue occurs if there are 1 to 19 tabs currently open.
In other words, it does not happen if there are 20 tabs and more.
This is because all thumbnails will be smaller from 20 tabs.
Therefore, it is possible to settle to the right space.

In conclusion, I believe that the bar with the + mark on the far right is the cause.
It would have been better to position it at the top, or allow the user to adjust the position arbitrarily.

**Comment by LichenSymbiont** [comment]
Having a whole bar for a single button... that's a design that will definitely not last (evolutionarily, hehe). Eighter use it, or if possible: add a special pop-up menu for right-clicking the background.
Or... did the old native panarama system allow you to just "draw" your own box to create a tab-group box? That's another solution.

It was the one thing that seemed out of place when first trying this addon, besides not having tab-group boxes being placed sanely when adding them.
